specification.md

```diff
@@ -120,11 +124,11 @@ Signature Standard FIPS PUB 186-4: https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.F
 
 ### 4.3.  Authentication
 
-API authentication for OpenGDPR requests is out of scope for this document, and is left to the Processor to implement. Callbacks **MUST** be authenticated by a digital signature issued by the certificate detailed in section 4.1.
+API authentication for OpenDSR requests is out of scope for this document, and is left to the Processor to implement. Callbacks **MUST** be authenticated by a digital signature issued by the certificate detailed in section 4.1.
 
 ## 5. Identities
 
-The identity types and schema documented below are reused throughout all OpenGDPR API contracts.
+The identity types and schema documented below are reused throughout all OpenDSR API contracts.
 
 ### 5.1.  Identity Type Keys
```

Comment:
I think this point should be more generic. At the moment it is very focused only on marketing and advertising. A processor could be anything from a simple hosting provider to a SaaS vendor, but also your lawyer or your accounting firm.